tests/behat/features/wip/homepage_edit-account-auth.feature

```diff
@@ -0,0 +1,60 @@
+@wip
```

Comment:
@FloQuinn is this meant to be in templates?

Reply:
I don't know what this is.  I am not going to create the page on the live site.